app/notifications/notifications_letter_callback.py

```diff
@@ -55,7 +50,5 @@ def process_letter_response():
 
         if filename.lower().endswith("rs.txt") or filename.lower().endswith("rsp.txt"):
             current_app.logger.info("DVLA callback: Calling task to update letter notifications")
-            update_letter_notifications_statuses.apply_async([filename], queue=QueueNames.NOTIFY)
```

Comment:
Should this whole method be removed?

Reply:
yes it should, but then the tests needs to be removed. So falling the principle I was using below - I didn't remove it.